app/controllers/streams_controller.rb

```diff
@@ -4,22 +4,14 @@
 # Streams API Controller
 #
 class StreamsController < ApplicationController
+  def initialize(params)
+    super(params)
+
+    file = File.read(File.join(File.dirname(__FILE__), '../../data/seedFile.json'))
```

Comment:
Nice one, we can make this seed_file coming from `models` layer. So `app/models/stream.rb`

Reply:
Also, for YML load there is a library that will handle read.

https://stackoverflow.com/questions/8878389/ruby-loading-config-yaml-file-in-same-dir-as-source/8878436

Reply:
I am not particularly clear on why we need to load the json file from yaml but I am going to pass on that one as it is not pretty clear to me at the moment.

Reply:
Was thinking of json as yml 🤦 , looks good as is.